.github/workflows/build-native.yml

```diff
@@ -1,7 +1,7 @@
 name: Build Native
 
 on:
-  workflow_dispatch
+  [workflow_dispatch, push, pull_request]
```

Comment:
Would be cool to communicate that pre-release packages are available. Maybe in readme.md?